### PR TITLE
ReviewDB: add more context menu shortcuts to view reviews

### PR DESCRIPTION
**Pull request by Sqaaakoi**
Small patch to add the "View Reviews" menu option to a few places

User profile modal ... menu
User profile modal ... menu screenshot

Guild context menu
Guild context menu screenshot

User context menu
User context menu screenshot

I made this mostly so I can click on a reviewers profile then see their reviews

### CODE REVIEW

File: src/plugins/reviewDB/index.tsx [hunk 1]
```diff
@@ -47,14 +47,29 @@ const guildPopoutPatch: NavContextMenuPatchCallback = (children, props: { guild:
     );
 };
 
+const userContextPatch: NavContextMenuPatchCallback = (children, props: { user?: User, onClose(): void; }) => {
```
Comment (Vendicated) [inline src/plugins/reviewDB/index.tsx:49]:
```suggestion
const userContextPatch: NavContextMenuPatchCallback = (children, { user }: { user?: User, onClose(): void; }) => {
```

then you dont need the non null assertions

Comment (Sqaaakoi) [reply src/plugins/reviewDB/index.tsx:49]: i don't understand how destructuring will fix that? if you look at the commit history you will see why the check was even implemented in the first place

Comment (Sqaaakoi) [reply src/plugins/reviewDB/index.tsx:49]: image

Comment (Sqaaakoi) [reply src/plugins/reviewDB/index.tsx:49]: ImageZoom also has a similar issue

Comment (Vendicated) [reply src/plugins/reviewDB/index.tsx]: i resolved it myself

**Comment by Vendicated** [comment]
very useful, thanks a lot!

**Pull request merged**